Makefile

```diff
@@ -0,0 +1,13 @@
+PHONY: default deps dev
```

Comment:
We might want to encode these commands as instructions in the `README.md` of the repository (as well as where to get Hugo) instead of in a `Makefile`.